mempool/clist_mempool.go

```diff
@@ -18,6 +19,10 @@ import (
 	"github.com/cometbft/cometbft/types"
 )
 
+const (
+	timeoutRecheck = 100 * time.Millisecond
```

Comment:
This appears to be pretty arbitrary.

Reply:
Changed to a config parameter. And the default value is now 1000ms.